### ISSUE TITLE
Wrong formatting in mintty.1

### ISSUE THREAD

**Issue by rillig** [mintty/mintty#1055]
In `mintty.1`, the formatting looks wrong in 2 places. The font continues to be non-plain after the `\fP`. I fixed it like this:

~~~diff
  default size
- \fICOLS\fBx\fIROWS\fP
+ \fICOLS\fP\fBx\fP\fIROWS\fP
  Window icon
- \fIFILE\fP[\fB,\fIINDEX\fP]
+\fIFILE\fP[\fB,\fP\fIINDEX\fP]
~~~

To me, it seems necessary to terminate each change of font immediately with a `\fP`. I don't understand the underlying model though since adding `\fP` 3 times at the end doesn't fix the formatting, therefore the font changes don't seem to be stacked.

The lines directly below a `.TP` macro don't seem to need this fix. There may be an implicit font reset in the macro. I'm not familiar enough with formatting manual pages, so I can just guess what really happens under the hood.

There are some more issues in the same file (maybe in other files as well), which I found using this command:
~~~sh
grep -E '\\fB[^\\]*\\fI|\\fI[^\\*]\\fB' mintty.1
~~~

**Comment by mintty** [comment]
I don't see an effect in these cases but I saw it in other cases and you're right, font embedding tags should be symmetric and not nested. Fixed also a few more cases.

**Comment by mintty** [comment]
Released 3.4.2.

**Comment by rillig** [comment]
Just for your reference, here are some screenshots, taken with mintty 3.2.0 on Cygwin, Windows 10, showing [mintty.1 before the commit that fixed the fonts](https://github.com/mintty/mintty/blob/14f2a32678b4f21f242ee28ddd48e510a250b936/docs/mintty.1):

The text after ThemeFile is bold but shouldn't be:
![image](https://user-images.githubusercontent.com/3233724/98180805-2d93f900-1f02-11eb-85a1-3196d3822cb3.png)

The closing bracket is bold but shouldn't be:
![image](https://user-images.githubusercontent.com/3233724/98180919-6e8c0d80-1f02-11eb-8c59-0578482b658b.png)

The word `MONITOR` should be italic (or underlined) but is bold:
![image](https://user-images.githubusercontent.com/3233724/98180980-8bc0dc00-1f02-11eb-8ce5-525f02fcd570.png)

After the fix, the `MONITOR` and everything else looks as intended:
![image](https://user-images.githubusercontent.com/3233724/98181169-f6721780-1f02-11eb-968e-6f1e79d79aa3.png)
Oh, wait, the nested brackes are bold. I'm not sure whether this is intended or not.
